packages/components/src/core/LoadingIndicator/index.stories.tsx

```diff
@@ -30,7 +30,7 @@ export const Default = {
 // Live Preview
 
 const LivePreviewDemo = (props: Args): JSX.Element => {
-  const spacings = defaultAppTheme?.spacing;
+  const spacings = sharedAppTheme?.spacing;
```

Comment:
Another spot where we want to see if we can consume the theme from something other than an import.

Reply:
Done!